Source/XUnitTest/GameSessionTest.cs

```diff
@@ -0,0 +1,61 @@
+﻿using LudoGameEngine;
+using System;
+using Xunit;
+
+namespace XUnitTest
+{
+    public class GameSessionTest
+    {
+        [Theory]
+        [InlineData(2)]
+        [InlineData(3)]
+        [InlineData(4)]
+        public void SetPlayerAmount_ValidAmount_True(int input)
+        {
+            GameSession gameSession = new GameSession
+            {
+                PlayerAmount = input
+            };
+
+            Assert.True(gameSession.PlayerAmount > 1 && gameSession.PlayerAmount < 5);
```

Comment:
I assume that the you expect what you out in?
I'm a bit unsure why this check (with the > && <) is needed
```c#
        [Theory]
        [InlineData(2)]
        [InlineData(3)]
        [InlineData(4)]
        public void SetPlayerAmount_ValidAmount_True(int input)
        {
            GameSession gameSession = new GameSession
            {
                PlayerAmount = input
            };

            Assert.Equals(input, gameSession.PlayerAmount);
```